libpod/container.go

```diff
@@ -169,6 +187,60 @@ func (c *Container) LogPath() string {
 	return c.logPath()
 }
 
+// ExitCode returns the exit code of the container as
+// an int32
+func (c *Container) ExitCode() (int32, error) {
+	c.lock.Lock()
+	defer c.lock.Unlock()
+	if err := c.runtime.state.UpdateContainer(c); err != nil {
```

Comment:
This can be changed to c.syncContainer()

Reply:
fixed